CHANGELOG.md

```diff
@@ -4,6 +4,82 @@
+- Fix crash when using `RecordingStream::set_thread_local` on macOS [#3929](https://github.com/rerun-io/rerun/pull/3929)
+- Add improved `spawn` function [#3996](https://github.com/rerun-io/rerun/pull/3996) [#4031](https://github.com/rerun-io/rerun/pull/4031)
+- Redesign `clap` integration [#3997](https://github.com/rerun-io/rerun/pull/3997) [#4040](https://github.com/rerun-io/rerun/pull/4040)
+- `RecordingStream`: introduce `connect_opts` [#4042](https://github.com/rerun-io/rerun/pull/4042)
+- Add `disable_timeline` function [#4068](https://github.com/rerun-io/rerun/pull/4068)
+
+#### 🪳 Bug Fixes
+- Fix grayscale images being too dark [#3999](https://github.com/rerun-io/rerun/pull/3999)
+- Prevent badly sized tensors from crashing the viewer [#4005](https://github.com/rerun-io/rerun/pull/4005)
+- Fix selection history right-click menu not working [#3819](https://github.com/rerun-io/rerun/pull/3819)
+
+#### 🌁 Viewer Improvements
+- Replace `--strict` flag with `RERUN_PANIC_ON_WARN` env-var [#3872](https://github.com/rerun-io/rerun/pull/3872)
+- Support NV12-encoded images [#3541](https://github.com/rerun-io/rerun/pull/3541) (thanks [@zrezke](https://github.com/zrezke)!)
+
+#### 🧑‍🏫 Examples
+- `--max-frame` support for tracking examples [#3835](https://github.com/rerun-io/rerun/pull/3835)
+
+#### 📚 Docs
+- Synchronize code examples and their screenshots [#3954](https://github.com/rerun-io/rerun/pull/3954)
+- Improve docs for `TextDocument` example [#4008](https://github.com/rerun-io/rerun/pull/4008)
+- Fix typos in documentation and code comments [#4061](https://github.com/rerun-io/rerun/pull/4061) (thanks [@omahs](https://github.com/omahs)!)
+
+#### 🖼 UI Improvements
+- Add basic support for in-app "Quick Start" guides [#3813](https://github.com/rerun-io/rerun/pull/3813) [#3912](https://github.com/rerun-io/rerun/pull/3912)
+- Add copy-button to markdown code blocks [#3882](https://github.com/rerun-io/rerun/pull/3882)
+- Add warning in the Quick Start guides about Safari breaking Copy to Clipboard [#3898](https://github.com/rerun-io/rerun/pull/3898)
+
+#### 🎨 Renderer Improvements
+- Add easy way to dump out final wgsl shader [#3947](https://github.com/rerun-io/rerun/pull/3947)
+
+#### 🧑‍💻 Dev-experience
+- Approve all workflow runs for a specific contributor PR [#3876](https://github.com/rerun-io/rerun/pull/3876)
+- Make codegen I/O-free and agnostic to output location [#3888](https://github.com/rerun-io/rerun/pull/3888)
+- Configure pytest to fail on warnings [#3903](https://github.com/rerun-io/rerun/pull/3903)
+- Improve `taplo` output on failure [#3909](https://github.com/rerun-io/rerun/pull/3909)
+- Automatically synchronize build.rerun.io & release assets [#3945](https://github.com/rerun-io/rerun/pull/3945)
+- New helper script to run fast lints and pre-push hook that runs it [#3949](https://github.com/rerun-io/rerun/pull/3949)
+- CI: Rerun CLI as a release asset [#3959](https://github.com/rerun-io/rerun/pull/3959)
```

Comment:
🤔 could be in the highlight section that we have now release artifacts on each release making it easier to get to the "raw stuff".
Annoying we don't get any telemetry from that yet I think